x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/flyout/index.tsx

```diff
@@ -63,6 +71,20 @@ export const EventFiltersFlyout: React.FC<EventFiltersFlyoutProps> = memo(
       },
     });
 
+    const isPlatinumPlus = useLicense().isPlatinumPlus();
+    const isEditMode = useMemo(() => type === 'edit' && !!id, [type, id]);
+    const [wasByPolicy, setWasByPolicy] = useState<boolean | undefined>(undefined);
+
+    const showExpiredLicenseBanner = useMemo(() => {
```

Comment:
`useMemo` is not needed here. you use more CPU calculating if it should memoized than calculating the final result of the variable.